source/server/overload_manager_impl.cc

```diff
@@ -72,15 +115,21 @@ Stats::Gauge& makeGauge(Stats::Scope& scope, absl::string_view a, absl::string_v
 
 OverloadAction::OverloadAction(const envoy::config::overload::v3::OverloadAction& config,
                                Stats::Scope& stats_scope)
-    : active_gauge_(
-          makeGauge(stats_scope, config.name(), "active", Stats::Gauge::ImportMode::Accumulate)) {
+    : state_(OverloadActionState::inactive()),
+      active_gauge_(
+          makeGauge(stats_scope, config.name(), "active", Stats::Gauge::ImportMode::Accumulate)),
+      scaling_gauge_(
+          makeGauge(stats_scope, config.name(), "scaling", Stats::Gauge::ImportMode::Accumulate)) {
```

Comment:
In the original PR I think we [decided](https://github.com/envoyproxy/envoy/pull/11697#commitcomment-40148417) to have a "scale_value" gauge (with a value ranging from 0-100) instead of this boolean gauge. The doc change you have in this PR also reflects that.